src/codegen/pass2_generate.cc

```diff
@@ -1781,9 +1784,16 @@ CodeList* gen_bitmap(Output& output, const CodeBitmap* bitmap, const std::string
     CodeList* stmts = code_list(alc);
 
     const std::string& name = bitmap_name(opts, cond);
-    text = opts->lang == Lang::C
-           ? o.cstr("static const unsigned char ").str(name).cstr("[] = {").flush()
-           : o.str(name).cstr(" := []byte{").flush();
+    switch (opts->lang) {
+        case Lang::C:
+            text = o.cstr("static const unsigned char ").str(name).cstr("[] = {").flush();
+            break;
+        case Lang::D:
+            text = o.cstr("immutable char[] ").str(name).cstr(" = {").flush();
+            break;
+        default:
+            text = o.str(name).cstr(" := []byte {").flush();
+    }
```

Comment:
Sorry, I saw that was in the diff but I didn't pay attention in which case. Can you remove the space as it was before, and also please spell out the cases (Go and RUST rather than just default case). No need to update test cases for unrelated languages.

Reply:
Sure thing